src/Sulu/Bundle/HttpCacheBundle/Tests/Unit/Cache/CacheManagerTest.php

```diff
@@ -36,7 +36,21 @@ public function setUp(): void
         $this->cacheManager = new CacheManager($this->fosCacheManager->reveal());
     }
 
-    public function testInvalidateTag()
+    public function testSupportsInvalidate(): void
+    {
+        // proxy client doesn't support tag invalidation
```

Comment:
not sure what this comment means? 😇

Reply:
that the prox client does not support the tag invalidation 😂  not sure how i should write that different 🙈

Reply:
but why does this matter for this test case? is the proxy client used? 🤔

Reply:
@wachterjohannes ?